tests/cloudformation/file_formats/cfn_utils/test_yaml.yaml

```diff
@@ -0,0 +1,30 @@
+AWSTemplateFormatVersion: "2010-09-09"
```

Comment:
@Saarett I think it might be better to place it in dir `tests/cloudformation/utils/` next to the test that uses it